js/main.js

```diff
@@ -1,6 +1,73 @@
 const menuIcon = document.querySelector('.menuIcon');
 const nav = document.querySelector('.navList');
 const linkedWord = document.querySelectorAll('.navWord');
+const modal = document.querySelector('.modals');
+const button = document.querySelectorAll('.cardBtn');
+const projectButton = document.querySelector('.seeButton');
+
+const projectCollection = [
```

Comment:
Pop-up on the desktop has design issues. Text on the right side is overlapping the buttons. There should be a proper margin between them. Please use the same font for the title as in the design. Kindly see the screenshot below and fix all the issues.
![desktop](https://user-images.githubusercontent.com/35504149/137355025-3a2b4507-f332-45c6-aabe-b8acf66896f1.png)